sql/sql_plugin_var.h

```diff
@@ -277,6 +277,16 @@ class sys_var_pluginvar : public sys_var {
   bool do_check(THD *thd, set_var *var);
   virtual void session_save_default(THD *, set_var *) {}
   virtual void saved_value_to_string(THD *thd, set_var *var, char *def_val);
+
+  /**
+    Set a PERSIST_ONLY value for a variable. The resulting value should be
+    set to a string representation of the actual value.
+
+    @param[in]     thd   Thread context.
+    @param[in]     var   Plugin variable.
+    @param[in,out] dest  Destination memory pointer.
+  */
+  void persist_only_update(THD *, set_var *var, char *dest);
```

Comment:
function name is misleading. I was thinking it is updating somehing. AFAIU, this is about geting string representation of variable value.  may be get_string() or to_string() is sufficient.

Reply:
Changed it according to Kamil's suggestion - "persist_only_to_string"

Reply:
Ok, looks better..